src/utils/telemetryUtils.ts

```diff
@@ -134,10 +158,11 @@ export class TelemetryErrorHandler implements ErrorHandler {
   }
 
   error(error: Error, message: Message, count: number): ErrorHandlerResult {
-    this.telemetry.send({
-      name: "ansible.lsp.error",
-      properties: { jsonrpc: message.jsonrpc, error: error.message },
+    sendTelemetry(this.telemetry, true, "ansible.lsp.error", {
```

Comment:
The second parameter is set to `true`. Does it mean that this telemetry event can be sent even if the telemetry service is not initialized?

Reply:
Have added `sendTelemetry` function to provide a single to call the telemetry send API. In the earlier code TelemetryErrorHandler call we are invoking the send method directly without checking for initiliased however there is a check added to ensure telemetryService object is initialised.

Reply:
Thanks for the explanation!